src/python/WMCore/MicroService/MSPileup/MSPileup.py

```diff
@@ -69,8 +69,14 @@ def getPileup(self, **kwargs):
 
         # check if filters are present and use it as projection fields
         projection = {}
-        for key in kwargs.get('filters', []):
-            projection[key] = 1
+        filters = kwargs.get('filters', [])
+        if type(filters) == str:
```

Comment:
Dennis, I would suggest the following variation to this code (but the result would be the same):
```
filters = kwargs.get('filters', [])
if isinstance(filters, str):
    filters = [filters]
for key in kwargs.get('filters', []):
                # filter out empty strings
etc etc
```